src/integration-test/java/uk/gov/hmcts/juror/api/moj/report/standard/UndeliverableListReportITest.java

```diff
@@ -0,0 +1,188 @@
+package uk.gov.hmcts.juror.api.moj.report.standard;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.test.context.jdbc.Sql;
+import uk.gov.hmcts.juror.api.moj.controller.reports.request.StandardReportRequest;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.StandardReportResponse;
+import uk.gov.hmcts.juror.api.moj.report.AbstractStandardReportControllerITest;
+import uk.gov.hmcts.juror.api.moj.report.ReportHashMap;
+import uk.gov.hmcts.juror.api.moj.report.ReportLinkedMap;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+
+@Sql({
+    "/db/truncate.sql",
+    "/db/mod/truncate.sql",
+    "/db/administration/createUsers.sql",
+    "/db/mod/reports/UndeliverableListReportITest_typical.sql"
+})
+@SuppressWarnings("PMD.LawOfDemeter")
+public class UndeliverableListReportITest extends AbstractStandardReportControllerITest {
+
+    @Autowired
+    public UndeliverableListReportITest(TestRestTemplate template) {
+        super(template, UndeliverableListReport.class);
+    }
+
+    @Override
+    protected String getValidJwt() {
+        return getBureauJwt();
+    }
+
+    @Override
+    protected StandardReportRequest getValidPayload() {
+        return addReportType(StandardReportRequest.builder()
+            .poolNumber("415230103")
+            .build());
+    }
+
+    @Test
+    @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
+    void positiveTypicalBureau() {
+        testBuilder()
+            .jwt(getBureauJwt())
```

Comment:
This is not needed as it will default to the value from getValidJwt

 @Override
    protected String getValidJwt() {
        return getBureauJwt();
    }

Reply:
Line 46 Removed